npe2/manifest/commands.py

```diff
@@ -92,9 +92,11 @@ class Config:
     def exec(
         self,
         args: tuple = (),
-        kwargs: dict = {},
+        kwargs: dict = None,
         _registry: Optional["CommandRegistry"] = None,
     ) -> Any:
+        if kwargs is None:
+            kwargs = {}
```

Comment:
The `kwargs: dict={}` pattern better seems better to me?

Reply:
that was a suggested refactor to avoid [mutable defaults](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).  but in this case it's probably fine